README.md

```diff
@@ -6,19 +6,23 @@ Most messages sent from the base chain to Fuel will use a predicate as the messa
 
 The Message to Contract Predicate is for messages that are trying to send a data payload to a designated Fuel contract. This predicate runs through the following checks:
 - Verify Inputs
-  - There is exactly one `InputCoin` and it has `asset_id` set as the base asset to cover gas costs
-  - There is exactly one `InputMessage` (the message with a data payload to send to a contract)
-  - There is at least one `InputContract` with `contractID` that matches the first 32 bytes in the message data field
-  - Additional `InputContract` are allowed but are not required
+  - There are no more than 8 total inputs
```

Comment:
It's not seen in this diff however there is a section below which states the version of `forc` that can be used. This version states a version that can be greater than 0.19.1 however given the number of breaking changes and how quickly releases come out it's unwise to allow an unlock version to be specified because the code might not work in a future version. The sway apps have to be bumped frequently to fix any breaking changes upon newer releases. The dependency should state the exact version rather than leaving it open ended for "anything greater than this". It's fine to manually test each version and state each version explicitly but not to leave it open ended.

Reply:
Agreed!